sdk/ml/azure-ai-ml/azure/ai/ml/_internal/entities/_artifact_cache.py

```diff
@@ -264,6 +269,9 @@ def get(
             # When the cache folder of artifact package exists, it's sure that the package has been downloaded.
             return artifact_package_path.absolute().resolve()
         if resolve:
+            check_sum_path = self._get_checksum_path(artifact_package_path)
+            if Path(check_sum_path).exists():
+                os.unlink(check_sum_path)
```

Comment:
what's the difference between unlink and remove?

Reply:
For the python doc, it said "unlink is semantically identical to remove".
https://docs.python.org/3/library/os.html#os.unlink
And this chat discusses unlink vs remove. It seems that they used to be identical, after some redone they became two separate functions
https://stackoverflow.com/questions/42636018/python-difference-between-os-remove-and-os-unlink-and-which-one-to-use

Reply:
Thx!